client/src/components/PatientSearchBar/PatientSearchBar.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
+import { useGetPatientsByNameLazyQuery } from '../../generated/graphql-types';
+import SearchBar from '../SearchBar/SearchBar';
+import PatientSearchBarProps from './PatientSearchBar.type';
+
+export default function PatientSearchBar({
+  handlePatientSelected
+}: PatientSearchBarProps) {
+  const [search, setSearch] = useState<string>('');
+  const [getPatientsByname, { data }] = useGetPatientsByNameLazyQuery();
+
+  const handleChange = (value: string): void => {
+    setSearch(value);
+  };
+
+  useEffect(() => {
+    if (search !== '') {
+      getPatientsByname({ variables: { search: search } });
```

Comment:
shouldn't we use a debounce here? so if dr Convergence has trouble writting a patient's name, it doesn't spam the server with queries

Reply:
in another PR ?